playgrounds/petclinic-maven-sb-2/src/main/resources/templates/fragments/layout.html

```
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+
+<html th:fragment="layout (template, menu)" xmlns:th="https://www.thymeleaf.org">
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)